bricks/template/__brick__/{{project_name.snakeCase()}}/ios/fastlane/Fastfile

```diff
@@ -102,6 +102,7 @@ platform :ios do
   # Firebase
   desc 'Build and upload Staging app to Firebase'
   lane :build_and_upload_staging_app do
+    bump_build
```

Comment:
set_app_version and bump_build have been called inside `build_and_upload_to_firebase`